docs/pages/database-access/reference/aws.mdx

```diff
@@ -3,17 +3,171 @@ title: Database Access AWS IAM Reference
 description: AWS IAM policies for Teleport database access.
 ---
 
+## Auto-discovery
 Teleport automatically discovers and configures IAM for RDS and Redshift given
 proper IAM permissions as described in the [AWS RDS & Aurora
 guide](../guides/rds.mdx) and the [AWS Redshift
 guide](../guides/postgres-redshift.mdx).
 
+The Teleport database service requires a set of AWS IAM permissions to:
+- update database configurations;
+- update its identity with connection-related permissions;
+
+You can generate and manage the permissions bootstrap command. Or if you prefer,
```

Comment:
I'm not sure it's clear what "the permissions bootstrap command" refers to in this sentence. The command is not introduced earlier, and I'm not sure a reader would know immediately what "the permissions bootstrap command" refers to unless they happen to have read another guide that mentions it more specifically.

I think we should add a short example of using the `teleport db configure bootstrap` command here.

Reply:
I've edited this paragraph, added the command reference link, and placed an example of how to execute it. It is not as complete as the description on the databases guides, but it is enough to get started with the command itself. Let me know if you feel that it is not enough.

Reply:
Thinking a little bit more about it, the explanation/examples of the bootstrap command might be out of this page scope since its primary purpose is to explain the permissions the database service requires, not how to create them. I think mentioning the command briefly and linking a guide would make more sense (similar to how we currently do with the policies) than simply explaining it. What do you think?